multimodal-search-tirg/flow-query.yml

```diff
@@ -5,22 +5,24 @@ with:
   rest_api: true
   port_expose: $JINA_PORT
 pods:
-  image_read:
+  read_pass_image:
```

Comment:
the image is not `pass` so no need to put `pass` I think

Reply:
We are reading and filtering only image in that pod. Shouldn't we keep pass 🤔

Reply:
we are actually `crafting` it, so for me is confusing compared to a `_pass` executor.

Reply:
How is `filter_and_read_image` or `filter_and_craft_image?

Reply:
I would not mention `fitler` , I would put `image_read`

Reply:
but not a big problem